File: src/main/java/com/partymakers/shareparty/application/friends/dto/FriendDescription.java
package com.partymakers.shareparty.application.friends.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FriendDescription {
    @JsonProperty("name")
    private String name;
    @JsonProperty("nickName")
    private String nickName;
    @JsonProperty("eMail")
    private String eMail;
}

File: src/main/kotlin/com/partymakers/shareparty/friends/presentation/dto/FriendDescription.kt
package com.partymakers.shareparty.friends.presentation.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class FriendDescription(
    @JsonProperty("name")
    val name: String,
    @JsonProperty("nickName")
    val nickName: String,
    @JsonProperty("eMail")
    val eMail: String
)